tests/conftest.py

```diff
@@ -180,7 +179,7 @@ def __init__(self, transport: httpx.ASGITransport) -> None:
 
         def handle_request(self, request: Request) -> Response:
             fut = loop.create_task(self.handle_async_request(request))
-            time_cutoff = time.time() + 1
+            time_cutoff = time.time() + 10
```

Comment:
I decided to increase this number because it doesn't make a real difference: fast things are going run fast anyway, but having a larger number does eliminate doubt regarding this parameter in case CI fails.